Automattic/vip-go-mu-plugins
# Update vip-support submodule to latest

## Description

Updates the vip-support submodule:

Bump Mustache lib for security
Remove unused REST endpoints
Fix unit test suite
Add ability to restrict support users by email address

## Changelog Description

### VIP Support: User Access Restrictions

Restricting which VIP Support accounts are allowed to access the site is now possible via the `VIP_SUPPORT_USER_ALLOWED_EMAILS` constant, which is an array of email addresses allowed to use the VIP Support User system.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. `git submodule update`
1. Add yourself as a support user
2. Log in with that support user and visit WP Admin
3. Should have regular permissions
4. In `vip-config/vip-config.php`, add `define( 'VIP_SUPPORT_USER_ALLOWED_EMAILS', array() );` (no users allowed)
5. Should have no permissions (but still logged in)
6. In `vip-config/vip-config.php`, change the constant to `define( 'VIP_SUPPORT_USER_ALLOWED_EMAILS', array( '$YOUREMAILHERE' ) );`
7. Should have regular permissions again
